lib/Vaultaire/Query.hs

```diff
@@ -248,29 +247,27 @@ eventMetrics :: (MonadIO m, MonadSafe m)
              -> TimeStamp
              -> Query m SimplePoint -- ^ result data point
 eventMetrics uri origin addr _start end
-  = Select $ readSimplePoints M.NoRetry uri addr (TimeStamp 0) end origin
+  = Select $ readSimplePoints uri addr (TimeStamp 0) end origin
 
 -- Raw Marquise queries -------------------------------------------------------
 
 readSimplePoints :: (MonadIO m, MonadSafe m)
-           => M.Policy
-           -> URI -> Address -> TimeStamp -> TimeStamp -> Origin
+           => URI -> Address -> TimeStamp -> TimeStamp -> Origin
            -> Producer SimplePoint m ()
-readSimplePoints pol uri a s e o = runMarquiseReader uri $ do
+readSimplePoints uri a s e o = runMarquiseReader uri $ do
   (MarquiseReader c) <- lift ask
-  hoist liftIO $  catchMarquiseAll (M.readSimplePoints pol a s e o c)
-                                   (\x -> lift $ M.logError $ show x)
-               >> return ()
+  void $ hoist liftIO $ catch
+       (M.readSimplePoints a s e o c)
+       (\(MarquiseException err) -> lift $ errorM "Vaultaire.Query.readSimplePoints" err)
```

Comment:
why catch? I've had user complaints that it should be left to them what to do with marquise exceptions, which is even more true now that there is no retry.